### PR TITLE
Add CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by RealRancor**
Shameless copied from https://github.com/owncloud/calendar/pull/839

Ref: https://github.com/owncloud/core/issues/14609

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+## Submitting issues
+
+If you have questions about how to install or use ownCloud, please direct these to the [mailing list][mailinglist] or our [forum][forum]. We are also available on [IRC][irc].
```
Comment (blizzz) [inline CONTRIBUTING.md]: We could also get an IRC channel of our own, #owncloud-bookmarks as the Mail app has (https://github.com/owncloud/mail/blob/master/CONTRIBUTING.md). What do you think?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+Help us to maximize the effort we can spend fixing issues and adding new features, by not reporting duplicate issues.
+
+[template]: https://raw.github.com/owncloud/core/master/issue_template.md
+[mailinglist]: https://mailman.owncloud.org/mailman/listinfo/owncloud
+[forum]: https://forum.owncloud.org/
+[irc]: https://webchat.freenode.net/?channels=owncloud&uio=d4
+
+### Contribute Code and translations
+Please check [core's contribution guidelines](https://github.com/owncloud/core/blob/master/CONTRIBUTING.md) for further information about contributing code and translations.
```
Comment (blizzz) [inline CONTRIBUTING.md]: Maybe add the note that the Contributor Agreement does not need to be signed for when only participating on bookmarks.

**Comment by blizzz** [comment]
My support for shameless actions! :rocket:

File: CONTRIBUTING.md (hunk 1 above)
Comment (RealRancor) [inline CONTRIBUTING.md]: Mhhh, the text is a generic text about getting help with the installation or usage of ownCloud so i think #owncloud fits?

Comment (blizzz) [inline CONTRIBUTING.md]: It's not different in mail. I think, if people look for ownCloud specifically, 
they will end up in core anyway. But to crush into bookmarks is for a reason.

**Comment by blizzz** [comment]
No real blocker → Merging

**Pull request merged**